Makefile

```diff
@@ -34,7 +34,7 @@ lint: lint-python lint-go
 .PHONY: vendor
 vendor:
 	@echo "Fetching dependencies..."
-	go mod vendor
+	cd management-service && go mod vendor
```

Comment:
We should probabl also add commands to vendor the `common` and `clients` modules before vendoring the management service.